react/components/Address.tsx

```diff
@@ -0,0 +1,25 @@
+import React, { FunctionComponent } from 'react'
+import { AddressRules, AddressSummary } from 'vtex.address-form'
+
+interface Props {
+  address: Address
+  pickup?: Parcel
+}
+
+const Address: FunctionComponent<Props> = ({ address, pickup }) => {
+  return (
+    <div
+      className="c-muted-1 mb5 mr10-m lh-copy"
```

Comment:
If that is exported, I would remove margins from it so it could be used in other places with more flexibility.

Reply:
Yep, just removed it :)
Will also do this to other components that are exported.